index.bs

```diff
@@ -113,6 +119,13 @@ directions, such as true north, or even non stationary directions, like in
 relation to a devices own z-position, drifting towards its latest most stable
 z-position.
 
+The data provided by the OrientationSensor API are similar to data from [=deviceorientation Event=], but the OrientationSensor
+API has some significant differences:
+1. The OrientationSensor API represents orientation data in WebGL-compatible formats (quaternion, rotation matrix).
+1. The OrientationSensor API satisfies stricter latency requirements.
+1. The OrientationSensor API defines exactly which [=low-level=] motion sensors are used to obtain the orientation data.
```

Comment:
I think you are missing one point, that with the old API there was differences between implementations, like whether they used magnetometer or not.